draft-ietf-rtcweb-jsep.xml

```diff
@@ -3619,10 +3619,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                     <t>Or, if the ICE ufrag and password values have
                     changed, trigger the ICE Agent to start an ICE
-                    restart and begin gathering new candidates for the
-                    media section, as defined in <xref target="RFC5245"
-                    />, Section 9.1.1.1, unless it has been marked as
-                    bundle-only.</t>
+                    restart and, unless it has been marked as
+                    bundle-only, begin gathering new candidates for the
+                    media section as described in <xref target="RFC5245"
+                    />, Section 9.1.1.1, and if this description is an
```

Comment:
Run-on sentence. Suggest splitting the answer processing out into its own sentence.